content/200-concepts/100-components/01-prisma-schema/03-generators.mdx

```diff
@@ -66,3 +66,6 @@ The following is a list of community created generators.
 
 - [DBML generator for Prisma](https://notiz.dev/blog/prisma-dbml-generator)
 - [Prisma documentation generator](https://github.com/pantharshit00/prisma-docs-generator)
+- [Prisma JSON Schema Generator](https://github.com/valentinpalkovic/prisma-json-schema-generator)
+- [Prisma Nestjs Graphql](https://github.com/unlight/prisma-nestjs-graphql)
+- [TypeGraphQL & Prisma 2 integration](https://github.com/MichalLytek/typegraphql-prisma)
```

Comment:
```suggestion
- [TypeGraphQL & Prisma integration](https://github.com/MichalLytek/typegraphql-prisma)
```
I think we refer to Prisma 2 as just "Prisma"

Reply:
Copy Paste 🙃